perses/samplers/multistate.py

```diff
@@ -29,11 +27,11 @@ def __init__(self, *args, hybrid_factory=None, **kwargs):
         self._hybrid_factory = hybrid_factory
         super(HybridCompatibilityMixin, self).__init__(*args, **kwargs)
 
-    def setup(self, n_states, temperature, storage_file, minimisation_steps=100,
```

Comment:
Did you say you had to _keep_ `minimisation_steps`, or is this just unused?

Reply:
You are right, this was not being used, but I think it doesn't hurt to have it back,  because it could actually break existing code if we don't.

Reply:
Yes, you are right. It wasn't being used but we should add it back since it _could_ break currently existing code.